CMakeLists.txt

```diff
@@ -432,6 +445,10 @@ if(GUNROCK_MGPU_TESTS)
   add_test(NAME TEST_MGPU_PR COMMAND pr market
     ${gunrock_INCLUDE_DIRS}/simple_example/bips98_606.mtx ${DEVICES} --undirected)
   set_tests_properties(TEST_PAGERANK PROPERTIES FAIL_REGULAR_EXPRESSION "INCORRECT")
+  
+  add_test(NAME TEST_MGPU_GRMAT COMMAND grmat market
+    ${gunrock_INCLUDE_DIRS}/simple_example/bips98_606.mtx ${DEVICES} --undirected)
+  set_tests_properties(TEST_GRMAT PROPERTIES FAIL_REGULAR_EXPRESSION "INCORRECT")
```

Comment:
We don't need GRMAT `ctests`, and there is no shared library implementation for it either. So, please remove this.

Reply:
I will make these changes, I had kept it just so that in future if we plan to add anything, it would be easier. But this can be done easily, I will remove it.

Reply:
Yeah, should be pretty easy, but we might end up using googletest instead of ctest for these kind of cases.

Reply:
I have made the changes and committed, please review